website/content/en/docs/building-operators/ansible/migration.md

```diff
@@ -23,15 +22,27 @@ part of the process to [Integrating Kubebuilder and Operator SDK][integration-do
 
 ### What is new
 
-Scaffolded projects now use:
+Projects are now scaffold using:
 
 - [kustomize][kustomize] to manage Kubernetes resources needed to deploy your operator
 - A `Makefile` with helpful targets for build, test, and deployment, and to give you flexibility to tailor things to your project's needs
 - Updated metrics configuration using [kube-auth-proxy][kube-auth-proxy], a `--metrics-addr` flag, and [kustomize][kustomize]-based deployment of a Kubernetes `Service` and prometheus operator `ServiceMonitor`
+- A preliminary support for plugins. For more info see the [extensible CLI and scaffolding plugins design document][plugins-phase1-design-doc]
+- A PROJECT file which stores more information about the resources in use, to enable plugins make useful decisions while scaffolding
+- Liveness and Readiness probes using [`healthz.Ping`][healthz-ping].
 
 ## How to migrate
 
-The easy migration path is to a project from the scratch and let the tool scaffold the new layout. Then, add your customizations and implementations. See below for an example.
+The easy migration path is to create a new project from the scratch and let the tool scaffold the files properly and then,
+just replace with your customizations and implementations. Following an example.
```

Comment:
```suggestion
The easy migration path is to initialize a new project, re-recreate APIs, then copy pre-v1.0.0 configuration files into the new project.
```

Reply:
-  I'd like to not use `pre-v1.0.0`. Also, it is described above anyway.  
- Then, people need to `let the tool scaffold the files properly` and not only ` re-recreate APIs`. 
-  Also, I am not sure if I agree with `configuration files into the new project.` It is about to copy the full implementation as well. Or the Ansible playbooks and roles are not a kind of implementation too? 

Also, I do not change this one, just copy and past from GO.

Reply:
>I'd like to not use `pre-v1.0.0`. Also, it is described above anyway

`pre-v1.0.0` is more informative than `just replace with your customizations and implementations`. Described where?

>Then, people need to `let the tool scaffold the files properly` and not only `re-recreate APIs`

What else do they need to do?

>Also, I am not sure if I agree with configuration files into the new project. It is about to copy the full implementation as well. Or the Ansible playbooks and roles are not a kind of implementation too

This is just supposed to be a general description of what needs doing. The specifics are discussed below.